static/js/brand-store/NotAuthorized/NotAuthorized.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import NotAuthorized from "./NotAuthorized";
+
+test("the 'is public' checkbox should not be checked when the current store is set to private", () => {
```

Comment:
There isn't actually a checkbox in this component....is this a copy/paste error?

Reply:
sorry - doing too many things at once! fixed